src/main/java/org/broadinstitute/hellbender/tools/copynumber/models/AlleleFractionLikelihoods.java

```diff
@@ -87,8 +86,7 @@ static double hetLogLikelihood(final AlleleFractionGlobalParameters parameters,
                 - n * log(majorFraction + minorFraction * lambda0RefMinor);
         final double refMinorLogLikelihood = logNotPi + logcRefMinor + Gamma.logGamma(rhoRefMinor) - rhoRefMinor * log(tauRefMinor);
 
-        final double outlierLogLikelihood = logPi + log10ToLog(log10Factorial(a) + log10Factorial(r) - log10Factorial(a + r + 1));
-
+        final double outlierLogLikelihood = logPi - FastMath.log(a + r + 1) - CombinatoricsUtils.binomialCoefficientLog(a+r,a);
```

Comment:
In principle, I support the removal of `log10Factorial`, but also note #7652. Although this expression is mathematically equivalent, numerically, it will result in a slight change in the output of ModelSegments---which has been stable for 3 years now.

Not a problem, but @tmelman should be aware of this potential change and plan accordingly, depending on what order the PRs are merged. And perhaps @droazen can make sure that minor numerical changes are mentioned in the release notes when appropriate.

Minor nitpick: can you fix the whitespace in the arguments of `binomialCoefficientLog`?

Reply:
Whitespace has been fixed.